### PR TITLE
Add db init scripts

### PR DESCRIPTION
**Pull request by jonathan-eckel**
I added an `init/` directory to show the code needed to parse a leveljson file and load it into a mysql database.  It uses poetry to install the python dependancies.  Sample credentials exist in these files to get the database up and going locally.

### CODE REVIEW

File: init/level-catalog-notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    "# multi file format\n",
+    "# data = []\n",
+    "# for fn in data_files:\n",
+    "#     with open(fn, \"r\") as fp:\n",
+    "#         level_data = json.load(fp)\n",
+    "#         data.append(level_data)\n",
+    "# \n",
+    "# levels = []\n",
+    "# for d in data:\n",
+    "#     levels += d[\"list\"]"
```
Comment (yinnie) [inline init/level-catalog-notebook.ipynb:137]: Should we uncomment this block for `multi file format`? Right now the notebook is executing on `data_files[0]` only I believe

File: init/run_sql.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+mysql catalog < ./sql/1-create-levels-table.sql
```
Comment (yinnie) [inline init/run_sql.sh:1]: so this is to load .sql into catalog db correct? and is supposed to be run inside the mysql 5.7 container.  Does it not need user info? `mysql -u root -p dots123! catalog < .....`

File: init/exec_docker_command.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker exec -it dots-level-catalog_db_1 `cat ./run_sql.sh`
```
Comment (yinnie) [inline init/exec_docker_command.sh:3]: should `dots-level-catalog_db_1` be `init_db_1` (name of container by default)? and works
I don't quite understand how `cat ./run_sql.sh` works here. I ran 
`docker exec -it init_db_1 sh` so I am in container shell session then ran `mysql -u root -p dots123! catalog < ./sql/1-create-levels-table.sql` and realized the `sql/1-create-levels-table.sql` file needs to be in `/mysql_data` to be shared with the container. right?

File: init/level-catalog-notebook.ipynb (hunk 1 above)
Comment (jonathan-eckel) [reply init/level-catalog-notebook.ipynb:137]: good catch!

File: init/exec_docker_command.sh (hunk 3 above)
Comment (jonathan-eckel) [reply init/exec_docker_command.sh:3]: same here, I'll remove this file

File: init/run_sql.sh (hunk 2 above)
Comment (jonathan-eckel) [reply init/run_sql.sh:1]: I think I can remove this, the python notebook handles this

**Pull request merged**